### ISSUE TITLE
Illegal property key ""

### ISSUE THREAD

**Issue by Nikerabbit** [SemanticMediaWiki/SemanticMediaWiki#920]
```
2015-03-25 07:18:00 translatewiki.net translatewiki_net-bw_: [543a430f] /w/i.php?title=Special:ExportRDF/Category:User_bcc&xmlmime=rdf   SMW\InvalidPropertyException from line 77 of /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/dataitems/SMW_DI_Property.php: Illegal property key "".
#0 /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/export/SMW_Exporter.php(179): SMW\DIProperty->__construct('')
#1 /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/export/SMW_Exporter.php(111): SMWExporter::makeExportDataForSubject(Object(SMW\DIWikiPage))
#2 /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/export/SMW_ExportController.php(143): SMWExporter::makeExportData(Object(SMW\SemanticData))
#3 /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/export/SMW_ExportController.php(433): SMWExportController->serializePage(Object(SMW\DIWikiPage), 0)
#4 /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/specials/SMW_SpecialOWLExport.php(189): SMWExportController->printPages(Array, 0, '19700101000000')
#5 /srv/mediawiki/tags/2015-03-03_12:42:07/extensions/SemanticMediaWiki/includes/specials/SMW_SpecialOWLExport.php(49): SMWSpecialOWLExport->exportPages(Array)
#6 /srv/mediawiki/tags/2015-03-03_12:42:07/includes/specialpage/SpecialPage.php(383): SMWSpecialOWLExport->execute('Category:User_b...')
#7 /srv/mediawiki/tags/2015-03-03_12:42:07/includes/specialpage/SpecialPageFactory.php(581): SpecialPage->run('Category:User_b...')
#8 /srv/mediawiki/tags/2015-03-03_12:42:07/includes/MediaWiki.php(270): SpecialPageFactory::executePath(Object(Title), Object(RequestContext))
#9 /srv/mediawiki/tags/2015-03-03_12:42:07/includes/MediaWiki.php(573): MediaWiki->performRequest()
#10 /srv/mediawiki/tags/2015-03-03_12:42:07/includes/MediaWiki.php(424): MediaWiki->main()
#11 /srv/mediawiki/tags/2015-03-03_12:42:07/index.php(46): MediaWiki->run()
#12 {main}
```
SMW version 2.1.1.

**Comment by mwjames** [comment]
Can you check whether #882 fixes your problem (scheduled for 2.1.2)?

**Comment by Nikerabbit** [comment]
I tested using SMW master and the error does not appear with it.

**Comment by mwjames** [comment]
#882 fixes the issue.

Follow-up, add a `rdf-003-category.json` integration test to cover the export of a category and til the test is deployed, we leave the issue open.

**Comment by mwjames** [comment]
Added test 0d99a5e, the issue itself was fixed by #882. The fix will be part of the 2.1.2 release.
